base.py

```diff
@@ -647,32 +647,69 @@ def invoke(self, context, event):
 
 bpy.utils.register_class(MBDynSetInstallPath)
 
-class MBDynRunSimulation(bpy.types.Operator):
-    """docstring for MBDynRunSimulation"""
+class MBDynSelectMbd(bpy.types.Operator, ImportHelper):
+    """docstring for MBDynSelectMbd"""
+
     bl_idname = "sel.mbdyn_mbd_file"
-    bl_label = "Select MBDyn mbd file"
+    bl_label = "Select MBDyn .mbd file"
```

Comment:
just call it "MBDyn input file", ".mbd" is a common but not required extension (it literally could be anything)